plugins/MauticOutlookBundle/Config/config.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * @package     Mautic
+ * @copyright   2014 Mautic Contributors. All rights reserved.
+ * @author      Mautic
+ * @link        http://mautic.org
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+return array(
+    'name'        => 'Outlook',
+    'description' => 'Enables integrations with Outlook for email tracking',
+    'version'     => '1.0',
+    'author'      => 'Werner Garcia',
+
+    'routes'   => array(
+        'public' => [
+            'mautic_outlook_tracker'             => [
+                'path'       => '/outlook/tracking.gif',
+                'controller' => 'MauticOutlookBundle:Public:trackingImage'
+            ]
+        ]
+    ),
+
+    'services'    => array(
```

Comment:
Are these used? I'm not seeing these actually used anywhere. This `MauticPlugin\MauticOutlookBundle\EventListener\FormSubscriber`,  `MauticPlugin\MauticOutlookBundle\Form\Type\FormFieldsType` seem to be registering a post form submit action that will show up when building a form but they don't seem to be doing anything. Let's remove them.